dotfiles/waybar/modules.json

```diff
@@ -325,7 +325,7 @@
         "tooltip-format-ethernet": " {ifname}\nIP: {ipaddr}\n up: {bandwidthUpBits} down: {bandwidthDownBits}",
         "tooltip-format-disconnected": "Disconnected",
         "max-length": 50,
-        "on-click": "alacritty --class dotfiles-floating -e nmtui",
+        "on-click": "$TERM --class dotfiles-floating -e nmtui",
```

Comment:
Can you replace $TERM with $(cat ~/dotfiles/.settings/terminal.sh). Then it will use the settings from the ML4W Dotfiles App (Tab System)